src/arrayMethodSort.js

```diff
@@ -3,9 +3,43 @@
 /**
  * Implement method Sort
  */
+const compareAsString = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  }
+
+  if (stringA < stringB){
+    return -1;
+  }
+
+  if (stringA === stringB) {
+    return 0;
+  }
```

Comment:
if 2 previous cases were not passed it means that stringA === stringB already
```suggestion
    return 0;
```